docs/resiliency/index.asciidoc

```diff
@@ -118,9 +118,10 @@ in the case of each type of failure. The plan is to have a test case that valida
 [discrete]
 === Run Jepsen (STATUS: ONGOING)
 
-We have ported the known scenarios in the Jepsen blogs that check loss of acknowledged writes to our testing infrastructure.
-The new tests are run continuously in our testing farm and are passing. We are also working on running Jepsen independently to verify
-that no failures are found.
+We have ported the known scenarios in the Jepsen blogs that check loss of
+acknowledged writes to our testing infrastructure. The new tests are run
+continuously in our testing farm and are passing. We will also monitor for new
+failure scenarios and adapt our test suite as needed.
```

Comment:
Is it still fair to say that the status of "Run Jepsen" is "ongoing", if we have no real intention of doing it (AFAICT)?

Reply:
Eh we could probably change this heading. It's fair to say that we monitor for new failure scenarios, whether they arise from new Jepsen tests or from some other source.